walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,118 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.Iterator;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    ArrayList<TimeRange> collection = new ArrayList<TimeRange>();
+    ArrayList<TimeRange> optionalCollection = new ArrayList<TimeRange>();
+    TimeRange timeRange = TimeRange.fromStartDuration(0, 1440);
+    collection.add(timeRange); //original timeRange
+    if (request.getDuration() >= 1440) {
```

Comment:
Put `1440` in a constant at the top of the file so that 1. it can be maintained more easily since it's used in multiple places, and 2. the constant name explains what the number 1440 represents